ganttproject/src/net/sourceforge/ganttproject/gui/taskproperties/TaskAllocationsPanel.java

```diff
@@ -78,7 +80,7 @@ public JPanel getComponent() {
     CommonPanel.setupComboBoxEditor(getTable().getColumnModel().getColumn(4), myRoleManager.getEnabledRoles());
 
     JPanel tablePanel = CommonPanel.createTableAndActions(myTable, myModel);
-    String layoutDef = "(ROW weight=1.0 (LEAF name=resources weight=0.5) (LEAF name=cost weight=0.5))";
+    String layoutDef = "(ROW weight=1.0 (LEAF name=resources weight=0.5) (COLUMN weight=0.5 (LEAF name=cost weight=0.5) (LEAF name=load weight=0.5)))";
```

Comment:
Please change the weights of cost and load to 0.1 -- this will group them in the upper part of the column closer to each other